.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Release Checklist
+about: items to be completed for each release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Confirm anything outstanding for release with other maintainers on #besu-release
+- [ ] Update changelog if necessary
+- [ ] Notify maintainers about updating changelog for in-flight PRs 
+- [ ] Optional: for hotfixes, create a release branch and cherry-pick, e.g. `release-<version>-hotfix`
+- [ ] Optional: create a PR against the hotfix branch to see the CI checks pass
+- [ ] On the appropriate branch/commit, create a calver tag for the release candidate, format example: 24.4.0-RC2
+- [ ] Create a DRAFT release using the new tag (careful not to publish!)
+- [ ] Draft homebrew PR
+- [ ] Draft documentation release
+- [ ] Sign-off with team
+- [ ] Start burn-in using the new <version-RCX> tag
+- [ ] Sign off burn-in
+- [ ] Convert the DRAFT into a PRE-RELEASE
+    - this is now public and notifies subscribed users
+    - publishes artefacts and version-specific docker tags
```

Comment:
I have removed this step from the WIP checklist as it would result in creating redundant 24.5.0-RC1 artefacts.

See https://github.com/hyperledger/besu/pull/6963#issuecomment-2065575149 for context.

Most of the time we will want to go straight from draft/burn-in -> full release, with pre-release being a necessary in-between step for now. That pre-release needs to use the full release tag e.g. 24.5.0 instead of 24.5.0-RC1.